posts/2024-11-27-Rust-2024-public-testing.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: "Rust 2024 call for testing"
+author: "Eric Huss & TC"
+team: the Edition 2024 Project Group <https://doc.rust-lang.org/nightly/edition-guide/rust-2024/index.html>
+---
+
+# Rust 2024 call for testing
+
+We've been hard at work on Rust 2024. We're thrilled about how it has turned out. It's going to be largest edition since Rust 2015. It has a great many improvements that make the language more consistent and ergonomic, that further upon our relentless commitment to safety, and that will open the door to long-awaited features such as `gen` blocks, `let` chains, and the never (`!`) type. For more on the changes, see the nightly [Edition Guide](https://doc.rust-lang.org/nightly/edition-guide/rust-2024/index.html).
+
+As planned, we recently merged the feature-complete Rust 2024 edition [to the release train](https://github.com/rust-lang/rust/pull/133349) for Rust 1.85. It has now entered **nightly beta**.
```

Comment:
Maybe it's just me, but this is my first time seeing the term "nightly beta". I realized it is already used in the edition 2024 schedule in edition book, though still a bit confused and can't get what that means.

Reply:
presumably it means beta in the sense of"pre-release testing", not the beta branch of rust. I do think the term is a bit strange, and probably could be replaced, though.

Reply:
Thanks.  I added a footnote about this.